ChangeLog.rst

```diff
@@ -1,3 +1,88 @@
+2017-04-27 azure 2.0.0rc7
++++++++++++++++++++++++++
+
+**Disclaimer**
+
+Starting with this release, we fixed several packaging issues to improve our package generation system
+- Uninstalling of packages using "pip uninstall" now work properly (#1150)
+- setuptools install of "azure" works properly (#728)
+
+Update from 2.0.0rc6 *might* work, but we strongly suggest to create a fresh virtual environment.
+
+**Features**
+
+***CLI configuration***
+
+You can now execute your SDK script using the configuration of the `AzureCLI <https://github.com/Azure/azure-cli>`__
+
+Example:
+
+    from azure.common.client_factory import get_client_from_cli_profile
+    from azure.mgmt.compute import ComputeManagementClient
+    client = get_client_from_cli_profile(ComputeManagementClient)
+    client.virtual_machines.get('rg', 'vm')
+
+Configuration from the CLI is:
+- `az login <https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli>`__: access to credentials
+- `az account set --subscriptions <https://docs.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli>`__: access to subscription_id
+- `az cloud set --name <https://docs.microsoft.com/en-us/cli/azure/cloud#set>`__: access to base_url (sovereign cloud, Government, Germany, etc.)
+
+Note: CLI and SDK have different versionning story. We do recommend using two different virtual environment. Information will be shared
+automatically since the CLI saves them in the $HOME folder. You do require to install the `azure-cli-core` package in your SDK environment to use this feature.
+
+***Multiple ApiVersion support***
```

Comment:
If I understand correctly what this means, I think you should use "API version" rather than "ApiVersion" throughout this section. The latter looks like a Python class name, which is misleading.